File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.js
class Solution {
    removeDuplicates(nums){
    let k=1; 
        for(let n=1; n < nums.length;n++){
            if(nums[n]!=nums[n-1]){
                nums[k++]=nums[n];
         
            }
        }
        return k;
    }
}

File: Data Structures & Algorithms/rotate-array/submission-1.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        n = len(nums)
        k %= n
        while k:
            tmp = nums[n - 1]
            for s in range(n - 1, 0, -1):
                nums[s] = nums[s - 1]
            nums[0] = tmp
            k -= 1